pkg/systemd/logs.js

```diff
@@ -414,26 +419,40 @@ $(function() {
         }
     }
 
+    function split_search(text) {
+        let last_i = 0;
+        const words = [];
+        if (text.length - 1 !== " ")
```

Comment:
I'm sure that's supposed to mean `text[text.length - 1]`. This could use a comment why the string always has to end in a space (it's not obvious).

This should also handle the empty string without crashing.

Reply:
oops, of course. Thanks for spotting, fixed.